docs/source/building/windows.rst

```diff
@@ -15,31 +15,26 @@ Install dependencies:
    .. code-block:: bash
 
       pacman -S \
-        base-devel \
-        cmake \
-        diffutils \
-        gcc \
         git \
-        make \
-        mingw-w64-x86_64-binutils \
-        mingw-w64-x86_64-boost \
-        mingw-w64-x86_64-cmake \
-        mingw-w64-x86_64-curl \
-        mingw-w64-x86_64-miniupnpc \
-        mingw-w64-x86_64-nodejs \
-        mingw-w64-x86_64-onevpl \
-        mingw-w64-x86_64-openssl \
-        mingw-w64-x86_64-opus \
-        mingw-w64-x86_64-toolchain
+        mingw-w64-ucrt-x86_64-boost \
+        mingw-w64-ucrt-x86_64-cmake \
+        mingw-w64-ucrt-x86_64-cppwinrt \
+        mingw-w64-ucrt-x86_64-curl \
+        mingw-w64-ucrt-x86_64-miniupnpc \
+        mingw-w64-ucrt-x86_64-nodejs \
+        mingw-w64-ucrt-x86_64-onevpl \
+        mingw-w64-ucrt-x86_64-openssl \
+        mingw-w64-ucrt-x86_64-opus \
+        mingw-w64-ucrt-x86_64-toolchain \
 
 Build
 -----
 .. attention:: Ensure you are in the build directory created during the clone step earlier before continuing.
 
 .. code-block:: bash
 
-   cmake -G "MinGW Makefiles" ..
-   mingw32-make -j$(nproc)
+   cmake -GNinja ..
+   ninja
```

Comment:
Is using ninja a requirement? I'd prefer not to change this if possible.

Switching to ninja could be considered, but it should probably done in a separate PR and across all platforms simultaneously.

Reply:
For some reason, it's a dependency of the ucrt64 cmake package. It's used by default when Visual Studio configures CMake projects, too. 

I'll replace this with make and see how it goes.